infrastructure/helm/quantumserverless/values-ibm.yaml

```diff
@@ -73,7 +67,7 @@ gateway:
 # Redis configs
 # ===================
 
-redisEnable: true
+redisEnable: false
```

Comment:
nit: in theory, we could delete all the other config options for components we disable (not sure if that's something you'd want to do though)

Reply:
yep, for me it sounds good 👍

Reply:
Something like this @psschwei ?: [b461c11](https://github.com/Qiskit-Extensions/quantum-serverless/pull/668/commits/b461c1118c5258b0b77debb7f748808e2460e0cd)

Reply:
yes!